<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Plot</class>
 <widget class="QMainWindow" name="Plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: balck;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>[</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="xMin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(37, 37, 37);
	border: 1px black;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="yMax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(37, 37, 37);
	border: 1px black;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>y ∈</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="xMax">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(37, 37, 37);
	border: 1px black;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <widget class="QCustomPlot" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="yMin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(37, 37, 37);
	border: 1px black;
	color: white;
}</string>
      </property>
      <property name="text">
       <string>-10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>x ∈</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>[</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="7">
     <widget class="QLineEdit" name="FunEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: rgb(37, 37, 37);
	border: 1px black;
	color: white;
}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>f(x) = </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="ButtonPlot">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: black;
	border: 1px #ffffff;
	padding: 5px;
}

QPushButton: pressed {
	background-color: #808080;
	border: 1px #ffffff;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QPushButton" name="ButtonAuto">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: black;
	border: 1px #ffffff;
	padding: 5px;
}

QPushButton: pressed {
	background-color: #808080;
	border: 1px #ffffff;
	padding: 5px;
}</string>
      </property>
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMod">
    <property name="title">
     <string>Mod</string>
    </property>
    <addaction name="actionCalc"/>
    <addaction name="actionPlot"/>
    <addaction name="actionCredit"/>
   </widget>
   <addaction name="menuMod"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCalc">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calc</string>
   </property>
  </action>
  <action name="actionPlot">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </action>
  <action name="actionCredit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../qcustomplot-source/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
